files/install-hugo.yaml

```diff
@@ -3,11 +3,10 @@
   hosts: all
   tasks:
     - name: Install hugo
-      ansible.builtin.dnf:
-        name:
-          - hugo
-        state: latest
-      become: true
+      ansible.builtin.unarchive:
+        src: https://github.com/gohugoio/hugo/releases/download/v0.98.0/hugo_extended_0.98.0_Linux-64bit.tar.gz
+        remote_src: true
+        dest: /usr/bin/
```

Comment:
Btw, don't you need `become: true`, here? Since it's in `/usr/bin/`?

Reply:
yup, that was it, thanks!